pubsublite/pscompat/settings.go

```diff
@@ -95,6 +99,11 @@ type PublishSettings struct {
 	// Optional custom function that transforms a pubsub.Message to a
 	// PubSubMessage API proto.
 	MessageTransformer PublishMessageTransformerFunc
+
+	// Optional function to log informational messages that may be useful for
+	// debugging. For example, this can be set to log.Println. If not set,
+	// messages will not be logged.
+	OnLog LogFunc
```

Comment:
OnFoo funcs are typically invoked when "foo" happens. when does "log" happen? here, this should either be something like "OnError" (invoked when something goes wrong), or named more directly like "Logger".